### ISSUE TITLE
Can't send creditmemo email with grouped product

### ISSUE THREAD

**Issue by AurelienLavorel** [magento/magento2#17916]
### Preconditions
1. Any Magento 2 version

### Steps to reproduce
1. Create an order with grouped product
2. Make a creditmemo
3. Send email

### Expected result
1. Email is sent

### Actual result
```
[21-Jun-2018 10:37:43 UTC] PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Magento\Sales\Block\Order\Email\Items\Order\DefaultOrder::getItemPrice() must be an instance of Magento\Sales\Model\Order\Item, instance of Magento\Sales\Model\Order\Creditmemo\Item given, called in /var/www/html/vendor/magento/module-sales/view/frontend/templates/email/items/creditmemo/default.phtml on line 34 and defined in /var/www/html/vendor/magento/module-sales/Block/Order/Email/Items/Order/DefaultOrder.php:99
Stack trace:
#0 /var/www/html/vendor/magento/module-sales/view/frontend/templates/email/items/creditmemo/default.phtml(34): Magento\Sales\Block\Order\Email\Items\Order\DefaultOrder->getItemPrice(Object(Magento\Sales\Model\Order\Creditmemo\Item))
#1 /var/www/html/vendor/magento/framework/View/TemplateEngine/Php.php(59): include('/var/www/html/v...')
#2 /var/www/html/vendor/magento/framework/View/Element/Template.php(270): Magento\Framework\View\TemplateEngine\Php->render(Object(Magento\GroupedProduct\Block\Order\Email\Items\Order\Grouped), '/var/www/html/v...', Array)
#3 /var/www/html/vendor/magento/framework/View/Element/Template.php(300): Magento\Framework\View\Element\Template->fetchView('/var/www/html/v...')
#4 /var/www/html/vendor/magento/module-grouped-product/Block/Order/Email/Items/Order/Grouped.php(35): Magento\Framework\View\Element\Template->_toHtml()
#5 /var/www/html/vendor/magento/framework/View/Element/AbstractBlock.php(667): Magento\GroupedProduct\Block\Order\Email\Items\Order\Grouped->_toHtml()
#6 /var/www/html/vendor/magento/module-sales/Block/Items/AbstractItems.php(89): Magento\Framework\View\Element\AbstractBlock->toHtml()
#7 /var/www/html/vendor/magento/module-sales/view/frontend/templates/email/creditmemo/items.phtml(34): Magento\Sales\Block\Items\AbstractItems->getItemHtml(Object(Magento\Sales\Model\Order\Creditmemo\Item))
#8 /var/www/html/vendor/magento/framework/View/TemplateEngine/Php.php(59): include('/var/www/html/v...')
#9 /var/www/html/vendor/magento/framework/View/Element/Template.php(270): Magento\Framework\View\TemplateEngine\Php in /var/www/html/vendor/magento/module-sales/Block/Order/Email/Items/Order/DefaultOrder.php on line 99
```
I try to fix it here: https://github.com/magento/magento2/pull/16291 but it was refused.

**Comment by magento-engcom-team** [comment]
Hi @AurelienLavorel. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@AurelienLavorel do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by ghost** [comment]
Hi @AurelienLavorel  i'm able to reproduce following steps you described, please provide more information about steps to reproduce

**Comment by AurelienLavorel** [comment]
@magento-engcom-team give me test instance

**Comment by ghost** [comment]
@AurelienLavorel you need to-> "give me 2.2-develop instance"

**Comment by AurelienLavorel** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @AurelienLavorel. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @AurelienLavorel, here is your [Magento instance](https://i-17916-2-2-develop.engcom.dev.magento.com).
Admin access: [https://i-17916-2-2-develop.engcom.dev.magento.com/admin](https://i-17916-2-2-develop.engcom.dev.magento.com/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by AurelienLavorel** [comment]
Hi @engcom-backlog-nazar, I can't reproduce the bug in test instance, we only have check payment method :(
Do you have a payment method with ability to refund online in your test environment? If yes, could you try to make an online refund?

**Comment by ghost** [comment]
@AurelienLavorel  Ok i will try again with paypal ExpressCheckout.

**Comment by ghost** [comment]
@AurelienLavorel Same with Paypal Express checkout, maybe this issue already fixed in 2.2-develop branch ?

**Comment by AurelienLavorel** [comment]
I got it on that branch on a customer project, I'll check why. Sorry for the lost time.
